packages/docs/src/vite-plugin-synergy-styles/toCem.ts

```diff
@@ -14,7 +16,13 @@ import type {
 const getTypesAsArray = (tag: Tag): string[] => tag.type
   .split('|')
   .map(t => t.trim())
-  .map(t => `'${tag.name}--${t}'`);
+  .map(t => {
+    // If we don`t want a default value in a drop down, return an empty string
+    if (t === NO_DEFAULT) return '';
+
+    const separator = tag.name.includes('--') ? '-' : '--';
```

Comment:
Maybe we should add a comment here why it is needed. Otherwise we will read this line in 2 years (hopefully ;)) and don´t have a clue anymore. Mind adding some docs on this?

Reply:
I tried my best to explain it^^

Reply:
I ment the reason for `--` to `-` thingy. I think the NO_DEFAULT is fine as it is. However I don´t think we will be guessing why `tag.name.includes('--') ? '-' : '--';` is needed in a year or two from now.

Reply:
Yes. I added a comment with explanation on this line with the separator . My comment to your thread was meant for my commit with the new explanation  ;)

Reply:
:)